src/types.js

```diff
@@ -263,3 +263,27 @@ export type PmConversationData = {|
   recipients: string,
   unread: number,
 |};
+
+
+/**
+ * URL return type of parse-url
+ */
+export type UrlType={
+  slashes: boolean,
+  protocol: string,
+  hash: string,
+  query: {
+    realm: string,
+    email: string,
+    otp_encrypted_api_key: string
+  },
```

Comment:
This should probably be `query: { [key: string]: string }`.

(The existing definition file has `query: Object`, which isn't quite right.)

Reply:
I created the type for our particular use case. I will modify it to query: { [key: string]: string }`